desktop/core/src/desktop/js/reactComponents/FileChooserComponent/FileChooserComponentButton/FileChooserComponentButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './FileChooserComponentButton.scss';
+import FileChooserModal from '../FileChooserModal/FileChooserModal';
+import { Button } from 'antd';
+
+const FileChooserComponentButton = (): JSX.Element => {
```

Comment:
This component actually includes the whole FileChooserModal so calling it a button could be a bit misleading, for now perhaps we can call it `FileChooserWithButton` (there is no need to include "component" in any of the component or folder names).